guides/common/assembly_administering-hosts.adoc

```diff
@@ -12,11 +12,11 @@ include::modules/proc_editing-the-system-purpose-of-a-host.adoc[leveloffset=+1]
 include::modules/proc_editing-the-system-purpose-of-multiple-hosts.adoc[leveloffset=+1]
```

Comment:
This (and the one above it) could be `katello,satellite`, since Katello provides system purpose-related features.

Reply:
Applied.